user_sync/app.py

```diff
@@ -371,28 +371,46 @@ def credentials():
     help="Stores all sensitive fields and updates configuration files, replacing plaintext values with keys")
 def store():
     """
-    Stores credentials in the configuration file
+    Stores secure credentials in the configuration file
     This is an automated process
     """
     credential_manager = CredentialManager()
-    click.echo("Using backend: " + credential_manager.keyring_name)
-    click.echo("You have called the store-credential command.")
+    credential_manager.store()
+    retrieve_config_credentials = credential_manager.retrieve_config_credentials()
+    for cred_identifier in retrieve_config_credentials:
+        click.echo(f"{cred_identifier} stored securely in configuration file.")
 
 
-@credentials.command(help="Retreives currently stored credentials under the username 'user_sync'.")
-@click.option('--revert', default=False, is_flag=True, help="Reverts configuration files to plaintext state.")
-def retrieve(revert):
+@credentials.command(help="Retrieves currently stored credentials under the username 'user_sync'.")
+def retrieve():
     """
     Retrieves credentials from the configuration files
     By default, just prints out the values stored for UST
-    If --revert is used, updates config files with actual plaintext data
-    This is an automated process
     """
     credential_manager = CredentialManager()
-    click.echo("Using backend: " + credential_manager.keyring_name)
-    if revert:
-        click.echo("reverting...")
-    click.echo("you have called the retrieve credential command")
+    retrieve_from_credman_keys = credential_manager.retrieve_from_credman_keys()
+
+    if retrieve_from_credman_keys is None:
```

Comment:
Don't raise here - let's allow the possibility that there are nothing stored, and print out a message along those lines